src/App.js

```diff
@@ -1,10 +1,26 @@
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
-import CalculatorUI from './Components/calculatorUI';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+// eslint-disable-next-line import/extensions
```

Comment:
- [ ] Rather than disable the Linter strive to fix them as this makes you more accustomed to the style guide of the tech.

Reply:
done